CIScripts/Test/Utils/IisTcpTestDockerImageBuild.Tests.ps1

```diff
@@ -10,4 +10,10 @@ Describe "Initialize-IisTcpTestDockerImage" {
     It "Builds iis-tcptest image" {
         { Initialize-IisTcpTestDockerImage -Session $Session } | Should Not Throw
     }
+    
+    It "iis-tcptest image appear in docker inspect" {
+        Invoke-Command -Session $Session {
+            { docker inspect iis-tcptest } | Should Not BeNullOrEmpty
+        }
```

Comment:
* Should should be outside `Invoke-Command`
* Please remove `{}` braces before `| Should` – they are only needed in `Should [Not] Throw`.
* This testcase doesn't actually make sure that the build has been done before – we *shouldn't* rely on ordering of `It` clauses. Possible solutions:
    * Merge these two `It`s
    * Convert the first `It` to `BeforeEach`

Reply:
We suggest the following procedure:
1. Remove existing image from docker.
2. Build fresh image.
3. Verify that image has been created.